# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../net-tools/if_tr.h.patch
# Copyright (C) 2014 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- net-tools-1.60/lib/tr.c.vanilla	2014-06-17 15:21:20.000000000 +0200
+++ net-tools-1.60/lib/tr.c	2014-06-17 15:22:04.000000000 +0200
@@ -20,7 +20,7 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <net/if_arp.h>
-#include <linux/if_tr.h>
+#include <netinet/if_tr.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <errno.h>
